src/caveexpress/server/entities/Water.cpp

```diff
@@ -46,9 +50,8 @@ void Water::onContact (b2Contact* contact, IEntity* entity)
 		_fixturePairs.insert(std::make_pair(fixtureB, fixtureA));
 
 	entity->setTouchingWater(true);
-	// play the sound only once per second
-	if (_lastSoundTime < _time + 1000) {
-		_lastSoundTime = _time;
+	if (_lastSoundDT > min_lastSoundDT) {
```

Comment:
sorry, I still don't see why this change is needed - what's wrong with the `_lastSoundTime < _time + 1000` check here? why must it be float?

Reply:
It doesn't work. It triggers many sounds while in water.
I just used float and added dt in update to be sure. I didn't check why _time doesn't work.